ui/load/load.go

```diff
@@ -213,6 +215,8 @@ func loadIcons() Icons {
 		ListGridIcon:             decredmaterial.NewImage(decredIcons["list_grid"]),
 		DecredSymbolIcon:         decredmaterial.NewImage(decredIcons["decred_symbol"]),
 		DecredSymbol2:            decredmaterial.NewImage(decredIcons["ic_decred02"]),
+		GovernanceActiveIcon:     decredmaterial.NewImage(decredIcons["governance_a"]),
```

Comment:
```suggestion
		GovernanceActiveIcon:     decredmaterial.NewImage(decredIcons["governance_active"]),
```
Why isn't this governance_active

also it seems two images were added `governance_a` and `governance_active`